src/custom/state/orders/utils.ts

```diff
@@ -66,24 +68,89 @@ export function classifyOrder(order: OrderMetaData | null): ApiOrderStatus {
   return 'pending'
 }
 
-export function getExecutedPrice<TQuote extends Currency, TBase extends Currency>(
-  order: OrderMetaData
-): Price<TQuote, TBase> {
-  return new Price<TQuote, TBase>(
-    order.sellToken as unknown as TQuote,
-    order.buyToken as unknown as TBase,
-    order.executedSellAmount.toString(),
-    order.executedBuyAmount.toString()
-  )
+export type GetLimitPriceParams = {
+  buyAmount: string
+  sellAmount: string
+  buyTokenDecimals: number
+  sellTokenDecimals: number
+  inverted?: boolean
 }
 
-export function getLimitPrice<TQuote extends Currency, TBase extends Currency>(order: Order): Price<TQuote, TBase> {
-  return new Price<TQuote, TBase>(
-    order.inputToken as TQuote,
-    order.outputToken as TBase,
-    order.sellAmount.toString(),
-    order.buyAmount.toString()
-  )
+// TODO: Use the SDK when ready
+/**
+ * Calculates order limit price base on order and buy/sell token decimals
+ * Result is given in sell token units
```

Comment:
Actually, price has no units.
Maybe it's just a matter of phrasing it:
Result is given in SellToken/BuyToken == BuyToken per SellToken; SellToken == Base, BuyToken == Quote

Reply:
Copy pasta from https://github.com/gnosis/gp-ui/blob/36eae4570aac573e71d909042d3d4b18adfbf5aa/src/utils/operator.ts#L168-L168

Doesn't matter too much I think we should use the SDK after.

Reply:
But totally agree. Price doesn't have units. Will just delete that comment

Reply:
Wonder who wrote this shitty comment?
Oh, it was me, of course.